### DISCUSSION TITLE
Sorting Forks of a repo

### DISCUSSION THREAD

**Discussion by matthargett** [community/community#19850] [Mobile]
In July 2022, the mobile UI gives a list of forks for a repo with no sorting or filtering capabilities. It would be much more useful if we could sort by:
* fork creation date
* most recent commit to fork (default)
* fork creator name

It would be very useful if tapping on a fork opened the branch the most recent commit was to, rather than the default branch for the repo. Most people fork, create a branch, and do work there. Many repos have very long lists of branches, so it's not practical to look at all of the branches in a fork looking for more recent commits than the upstream fork.

Thanks!
![image](https://user-images.githubusercontent.com/1550766/176967789-ebd82d4e-16af-4bf5-ab33-0d8fce22745f.jpeg)

**Comment by iskae** [comment]
Hi @matthargett 👋 

Thanks a bunch for reaching out, we really appreciate any feedback from our users ❤️ 

We already have an internal tracking issue for filtering/sorting repository screens 🎉 I have also added your feedback in there too. 
Please stay tuned for our future releases 😊 

Regarding your second portion of your request: I believe this would be a bit harder to achieve, since every repository needs to have a main branch that our users can branch off and merge into. Would it be useful if we offer sorting by most recent commits on branches view?

**Reply by matthargett** [reply]
Sorting by most recent commits on branches helps, if it tells me what the name of the branch is, or directly links me to that branch code/commits view.

**Comment by Elaws** [comment]
Hi @iskae, any news please ? It's 4 years later and unless I'm mistaken forks still can't be sorted by e.g. stars ? Thanks.

**Reply by iskae** [reply]
Hi @Elaws 👋 

Unfortunately we were not able to prioritize improve filtering and sorting for forks specifically, but we are planning to address these concerns as soon as possible. It is still on our backlog and we are hoping to prioritize this soon 🙇
